acceptance/restorer_test.go

```diff
@@ -274,6 +276,8 @@ func testRestorerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 					h.AssertStringContains(t, analyzedMD.RunImage.Reference, restoreRegFixtures.ReadOnlyRunImage+"@sha256:")
 					h.AssertEq(t, analyzedMD.RunImage.Image, restoreRegFixtures.ReadOnlyRunImage)
 					h.AssertEq(t, analyzedMD.RunImage.TargetMetadata.OS, "linux")
+					t.Log("does not return the digest for an empty image")
+					h.AssertStringDoesNotContain(t, analyzedMD.RunImage.Reference, restoreRegFixtures.ReadOnlyRunImage+"@sha256:03cbce912ef1a8a658f73c660ab9c539d67188622f00b15c4f15b89b884f0e10")
```

Comment:
Thank you for highlight that this is a special SHA, I didn't know it.
My advice is to have it stored inside a kind of constant variable with a nice name to describe it properly and improve the readability.
From the external point of view it's like a "magic number" and should be clarified.

Perhaps something like `const  SHA_EMPTY_IMAGE = "@sha256:03cbce912ef1a8a658f73c660ab9c539d67188622f00b15c4f15b89b884f0e10"`
To have something like:
```
h.AssertStringDoesNotContain(t, analyzedMD.RunImage.Reference, restoreRegFixtures.ReadOnlyRunImage+SHA_EMPTY_IMAGE)
```

Or if you prefer, have something like:
```
var readOnlyEmptyRunImage = restoreRegFixtures.ReadOnlyRunImage+SHA_EMPTY_IMAGE
```
and have something
```
h.AssertStringDoesNotContain(t, analyzedMD.RunImage.Reference, readOnlyEmptyRunImage)
```

What do you think?

Reply:
Thanks for the suggestion @dlion ❤️ I was slow to update this one but I'll keep it in mind for next time